src/lib/templating.ts

```diff
@@ -83,6 +83,10 @@ export class PipelineInterpolator implements StepMatcher<S.PipelineStep> {
     return { ...step };
   }
 
+  concatenate(step: Readonly<S.ConcatenateStep>) {
+    return { ...step };
```

Comment:
Wouldn't it be worth to make the `separator` field templatable?

Reply:
No it's ok not to make it templatable. Never saw such a edge case